utils/estimated_histogram.hh

```diff
@@ -73,40 +73,59 @@ struct estimated_histogram {
 
     int64_t _count = 0;
     int64_t _sample_sum = 0;
+    int64_t _min = 0;
+    int64_t _max = 0;
 
     estimated_histogram(int bucket_count = 90) {
 
         new_offsets(bucket_count);
         buckets.resize(bucket_offsets.size() + 1, 0);
     }
 
-    seastar::metrics::histogram get_histogram(size_t lower_bucket = 1, size_t max_buckets = 16) const {
+    /*
+     * \brief get an accumulative histogram
+     * When getting an accumulative histogram, you can set what is the first bucket to report
+     * what is the number of buckets
+     * and what is the ratio between bucket limits.
+     * Setting the bucket_ratio for 1 or less, will use the buckets as-is
+     */
+    seastar::metrics::histogram get_histogram(size_t lower_bucket = 1, size_t max_buckets = 20, double bucket_ratio = 1.44) const {
```

Comment:
I don't understand how 1.3 gives more accurate results than the original 1.2.

Reply:
And why we use 1.44 for the default

Reply:
We never use the original 1.2, up until now we use a factor of 2, which appears to be too high.
I pick sqrt(2) it seems to make sense, I can pick anything else including the maximum available.